Assets/_BForBoss/_Shaders/FullScreenShaders/M_HitEffect.mat

```diff
@@ -26,7 +26,7 @@ Material:
         m_Offset: {x: 0, y: 0}
     m_Ints: []
     m_Floats:
-    - _EmissionStrength: 0
+    - _EmissionStrength: 1
```

Comment:
Was this meant to be touched?

Reply:
Whenever the the VFX stuff is updated, the material changes. 
Will need to create a ticket to have a manager that handles resets on these materials so that these doesn't change all the time. 

this mat in particual is the health hud vfx